@here/harp-mapview-decoder/test/TileGeometryLoaderTest.ts

```diff
@@ -343,24 +340,11 @@ describe("TileGeometryLoader", function() {
             expect(mapView.taskQueue.numItemsLeft(TileTaskGroups.CREATE)).equal(0);
 
             expect(mapView.taskQueue.processNext(TileTaskGroups.CREATE)).equal(false);
-
-            await willEventually(() => {
```

Comment:
I had to remove this, because this condition is never reached, i.e. the `processTechniques` function is never called because the task is expired

Reply:
The test should still have passed with `expect(spyProcessTechniques.callCount).equal(0); `

But anyway its probably not necessary to test this as its supposedly coverede with the empty TaskQueue test.

Reply:
@FraukeF , good point, yes I thought that because they weren't finishing and that it was tested elsewhere, I could safely ignore it.

Looking at it, the only condition that isn't met is the `isFinished` condition is never true, I will check why.